### ISSUE TITLE
Erreur 503 du rate limiter, mauvais code ?

### ISSUE THREAD

**Issue by Lifeismana** [romulusFR/lifap5-backend-2019-2020#21]
Quand se fait rate limit, on a une 503 du serveur qui correspond à "Service Unavailable"
Mais est-ce que ça serait pas plus cohérent de renvoyer une erreur [429 Too Many Requests](https://developer.mozilla.org/fr/docs/Web/HTTP/Status/429) à la place de la 503 ?

**Comment by Lifeismana** [comment]
En plus de ça on pourrait renvoyé un [Retry-After](https://developer.mozilla.org/fr/docs/Web/HTTP/Headers/Retry-After) avec un peut d'aléatoire pour étaler les requêtes qu'on réessaye et ça résoudrait un peu #17

**Comment by Webcretaire** [comment]
Je suis d'accord que le code 429 est bien adapté, par contre je ne pense pas que le `Retry-After` soit une bonne idée, pour la même raison que j'ai expliquée dans #17 : vous encourager à mettre des délais artificiels dans le client pour contrer des défauts du backend est une **mauvaise** idée, surtout pour un projet scolaire dans lequel on essaye de vous encourager à faire du code propre, fonctionnel, etc.
